tgui/packages/tgui/interfaces/AppearanceChanger.js

```diff
@@ -0,0 +1,310 @@
+                  onClick={
+                    () => act('facial_hair', { facial_hair: s.facialhairstyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
+          )}
+          {!!change_head_markings && (
+            <LabeledList.Item label="Head markings">
+              {head_marking_styles.map(s => (
+                <Button
+                  key={s.headmarkingstyle}
+                  content={s.headmarkingstyle}
+                  selected={s.headmarkingstyle === head_marking_style}
+                  onClick={
+                    () => act('head_marking', { head_marking: s.headmarkingstyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
+          )}
+          {!!change_body_markings && (
+            <LabeledList.Item label="Body markings">
+              {body_marking_styles.map(s => (
+                <Button
+                  key={s.bodymarkingstyle}
+                  content={s.bodymarkingstyle}
+                  selected={s.bodymarkingstyle === body_marking_style}
+                  onClick={
+                    () => act('body_marking', { body_marking: s.bodymarkingstyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
+          )}
+          {!!change_tail_markings && (
+            <LabeledList.Item label="Tail markings">
+              {tail_marking_styles.map(s => (
+                <Button
+                  key={s.tailmarkingstyle}
+                  content={s.tailmarkingstyle}
+                  selected={s.tailmarkingstyle === tail_marking_style}
+                  onClick={
+                    () => act('tail_marking', { tail_marking: s.tailmarkingstyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
+          )}
+          {!!change_body_accessory && (
+            <LabeledList.Item label="Body accessory">
+              {body_accessory_styles.map(s => (
+                <Button
+                  key={s.bodyaccessorystyle}
+                  content={s.bodyaccessorystyle}
+                  selected={s.bodyaccessorystyle === body_accessory_style}
+                  onClick={
+                    () => act('body_accessory', { body_accessory: s.bodyaccessorystyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
+          )}
+          {!!change_alt_head && (
+            <LabeledList.Item label="Alternate head">
+              {alt_head_styles.map(s => (
+                <Button
+                  key={s.altheadstyle}
+                  content={s.altheadstyle}
+                  selected={s.altheadstyle === alt_head_style}
+                  onClick={
+                    () => act('alt_head', { alt_head: s.altheadstyle })
+                  }
+                />
+              ))}
+            </LabeledList.Item>
```

Comment:
Ditto

Reply:
I cant think of a pheasible way to do this, but youre welcome to improve it later